Userland/Libraries/LibC/assert.cpp

```diff
@@ -16,8 +16,6 @@
 
 extern "C" {
 
-extern bool __stdio_is_initialized;
```

Comment:
already declared in `sys/internals.h`, like `__heap_is_stable` which is also used below.